src/main/java/com/racingcar/game/Game.java

```diff
@@ -33,11 +38,17 @@ private void playOneRound() {
         }
     }
 
-    private int[] getRoundResult() {
-        return Arrays.stream(cars)
-                .mapToInt(car -> car.getDistance())
-                .toArray();
+    private RoundResult getRoundResult(List<T> cars) {
+        RoundResult result = new RoundResult();
+        cars.forEach((car) -> {
+            result.add(car);
+        });
+        return result;
     }
 
+    private List<Car> decideWinners(List<T> cars) {
+        int max = cars.stream().mapToInt((car) -> car.getDistance()).max().orElseThrow(() -> new NoSuchElementException("can not find max distance"));
```

Comment:
```suggestion
        int max = cars.stream()
                .mapToInt((car) -> car.getDistance()).max()
                .orElseThrow(() -> new NoSuchElementException("can not find max distance"));
```
스트림은 줄바꿈을 통해 가독성을 높일 수 있습니다 ^_^